### PR TITLE
Epilepsy path ranking

### PR DESCRIPTION
**Pull request by ben-heil**
Created a notebook using the global path ranking metric to analyze the all paths for ten compounds whose connections to epilepsy had high metapath importance scores.

### CODE REVIEW

File: 10.epilepsy_path_ranking.ipynb [hunk 1]
```diff
@@ -0,0 +1,449 @@
...
+    "result_df.head()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result_df.to_csv('all_paths.tsv.gz')"
```
Comment (dhimmel) [inline 10.epilepsy_path_ranking.ipynb]: Would be best to save sorted by `source, path_importance, metapath`.

File: 10.epilepsy_path_ranking.ipynb [hunk 2]
```diff
@@ -0,0 +1,449 @@
...
+    "# For row in top_metapaths\n",
+    "\n",
+    "result_df = None\n",
+    "\n",
+    "for index, row in tqdm.tqdm_notebook(top_metapaths.iterrows(), total=len(top_metapaths.index)):\n",
+    "    metapath_df = get_paths_for_metapath(metagraph, row)\n",
+    "    if result_df is None:\n",
+    "        result_df = metapath_df\n",
+    "    else:\n",
+    "        result_df = pandas.concat([metapath_df, result_df], ignore_index=True)"
```
Comment (dhimmel) [inline 10.epilepsy_path_ranking.ipynb]: greater efficiency probably to append each df to a list and then concat the list once having exited loop.

If it'd be cumbersome to rerun. Just save this advice for next time.

**Comment by ben-heil** [comment]
Added the requested changes

File: 10.epilepsy_path_ranking.ipynb [hunk 3]
```diff
@@ -329,99 +327,99 @@
...
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": null,
    "metadata": {},
    "outputs": [],
    "source": [
-    "result_df.to_csv('all_paths.tsv.gz')"
+    "result_df.to_csv('all_paths.tsv.gz', index=False)"
```
Comment (dhimmel) [inline 10.epilepsy_path_ranking.ipynb]: AHH, we must specify `sep='\t'` otherwise this is a CSV not a TSV, my pet peeve.

**Pull request merged**